src/dsl.ts

```diff
@@ -177,19 +177,6 @@ export class SnykApiCheckDsl implements ApiCheckDsl {
           ),
         );
       },
-      should: (statement, handler) => {
```

Comment:
I'd like to understand this removal here and below better. Does ShouldOrMust now have a default so that this is no longer necessary?

Reply:
Yes -- `should` never did anything different than `must` so began to deprecate it -- and I updated your DSL to align with that. 

I can see a use for warnings and errors being different, but if we want to bring it back let's do it with intention and have different behaviors planned